Word.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+class Word {
```

Comment:
по PSR открывающая скобка у класса должна находиться на новой строке
https://www.php-fig.org/psr/psr-12/

```suggestion
class Word
{
```